framework/resources/Functional/filter/pushdown/filter_pruning/plan.json

```diff
@@ -0,0 +1,26 @@
+{
+  "testId": "filter_pushdown_filter_pruning",
```

Comment:
I think we should make the test ids unique. I know the extensions help here, but if we want to test for duplicates later, these may show up as false negatives.

Reply:
That was unintentionally. Fixed now.